src/solver/utils/ortools_utils.cpp

```diff
@@ -294,43 +301,52 @@ bool solveAndManageStatus(MPSolver* solver, int& resultStatus, const MPSolverPar
 }
 
 MPSolver* ORTOOLS_ConvertIfNeeded(const std::string& solverName,
+                                  const std::string& solverParameters,
                                   const Antares::Optimization::PROBLEME_SIMPLEXE_NOMME* Probleme,
                                   MPSolver* solver)
 {
     if (solver == nullptr)
     {
         Antares::Optimization::ProblemSimplexeNommeConverter converter(solverName, Probleme);
-        return converter.Convert();
+        return converter.Convert(solverParameters);
     }
     else
     {
         return solver;
     }
 }
 
+static bool computeUseBasis(const int optimizationNumber, const OptimizationOptions& options)
+{
+    return (optimizationNumber == PREMIERE_OPTIMISATION && options.useBasisOptim1)
+           || (optimizationNumber == DEUXIEME_OPTIMISATION && options.useBasisOptim2);
+}
 template<class SourceT>
 static void transferBasis(std::vector<operations_research::MPSolver::BasisStatus>& destination,
                           const SourceT& source)
 {
     destination.resize(source.size());
     for (size_t idx = 0; idx < source.size(); idx++)
     {
-        destination[idx] = source[idx]->basis_status();
+        destination[idx] = source[idx]->assibasis_status();
```

Comment:
Google won't accept anything like that. They want conversion of statuses in `MPSolver::BasisStatus`.